wikimedia/wikipedia-ios
# T378491: Update Slides

**Phabricator:** 
https://phabricator.wikimedia.org/T378491

### Notes
* Numbers may be wrong, they will be updated before launch

### Test Steps
1. Navigate to YIR (usual set up to enable)
2. Scroll through slides, ensure each one has a gradient, and the proper item associated with it (image vs text)

### Screenshots/Videos

https://github.com/user-attachments/assets/cb475213-22ad-4b1e-a695-9068abb81e67
